code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -109,3 +109,35 @@
 	desc = "Makes your nice clawed, scaled, hooved, armored, or otherwise just awfully calloused feet immune to glass shards."
 	cost = 1
 	var_changes = list("flags" = NO_MINOR_CUT) //Checked the flag is only used by shard stepping.
+
+/datum/trait/snowymarch //Oh my god those names are all awful.
+	name = "Snowy March"
+	desc = "Makes it easier to traverse snowy terrain."
+	cost = 1
+	var_changes = list("snow_movement" = -1)//Lower is less slowdown.
+
+/datum/trait/snowstride
+	name = "Snowy Stride"
+	desc = "It makes little difference to you whether the ground is snow or hard plating."
+	cost = 2
+	var_changes = list("snow_movement" = -2)//Lower is less slowdown.
```

Comment:
These are both cool ideas but I'm suspicious of snow that only has -1 slowdown giving you a +1 speed boost or something from this. Which tbh isn't even that terrible, it's kind of a neat idea. Just curious if you've researched how that will work at all, and if that is how it works, you should alter the description to something that might sound better with that.

Reply:
I can attest that it does not make you faster on snow, besides this, we could make snow actually give more slowdown. Perhaps like with CM variable snow levels. Also honestly, that was the *good* description.